coral/src/app/features/approvals/acls/AclApprovals.test.tsx

```diff
@@ -0,0 +1,207 @@
+    environment: "2",
+    teamname: "Ospo",
+    topictype: "Producer",
+    aclIpPrincipleType: "IP_ADDRESS",
+    environmentName: "TST",
+    teamId: 1003,
+    requestingteam: 1003,
+    appname: "App",
+    username: "amathieu",
+    requesttime: "2023-01-10T13:19:10.757+00:00",
+    requesttimestring: "10-Jan-2023 13:19:10",
+    aclstatus: "created",
+    approver: undefined,
+    approvingtime: undefined,
+    aclType: "Producer",
+    aclResourceType: undefined,
+    currentPage: "1",
+    otherParams: undefined,
+    totalNoPages: "1",
+    allPageNos: ["1", ">", ">>"],
+    approvingTeamDetails:
+      "Team : Ospo, Users : muralibasani,josepprat,samulisuortti,mirjamaulbach,smustafa,aindriul,",
+  },
+];
+
+const mockGetAclRequestsForApproverResponse = transformAclRequestApiResponse(
+  mockedAclRequestsForApproverApiResponse
+);
+
+describe("AclApprovals", () => {
+  beforeAll(() => {
+    mockIntersectionObserver();
+  });
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe("Skeleton", () => {
+    beforeAll(() => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      //@ts-ignore
+      useQuerySpy.mockReturnValue({ data: [], isLoading: true });
+
+      customRender(<AclApprovals />, {
+        queryClient: true,
+        memoryRouter: true,
+      });
+    });
+
+    afterAll(() => {
+      cleanup();
+      useQuerySpy.mockRestore();
+    });
+
+    it("shows a skeleton table", () => {
+      const skeleton = screen.getByTestId("skeleton-table");
+
+      expect(skeleton).toBeVisible();
+    });
+  });
+
+  describe("DataTable", () => {
+    beforeAll(async () => {
+      mockGetAclRequestsForApprover.mockResolvedValue(
+        mockGetAclRequestsForApproverResponse
+      );
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      //@ts-ignore
+      customRender(<AclApprovals />, {
+        queryClient: true,
+        memoryRouter: true,
+      });
+
+      await waitForElementToBeRemoved(screen.getByTestId("skeleton-table"));
+    });
+
+    afterAll(() => {
+      cleanup();
+    });
+
+    it("shows one header row and two data rows", async () => {
+      const rows = screen.getAllByRole("row");
+
+      expect(rows).toHaveLength(3);
+      expect(rows[0]).toHaveTextContent("Principals/Usernames");
+      expect(rows[1]).toHaveTextContent("mbasani");
+      expect(rows[2]).toHaveTextContent("User:*");
+    });
+
+    it("renders (prefixed) in Topic cell when appropriate", async () => {
+      const prefixedCells = screen.getAllByText("(prefixed)");
+      const notPrefixedCells = screen.getAllByText("aivtopic1");
+
+      expect(prefixedCells).toHaveLength(1);
+      expect(notPrefixedCells).toHaveLength(1);
+    });
+
+    it("renders all values for cells who can have multiple values", async () => {
+      const cells = screen.getAllByRole("cell");
+
+      expect(
+        cells.filter((cell) => {
+          return cell.textContent === "mbasanimaulbach";
```

Comment:
The fact that this is the text content of the cell already shows that this is what e.g. screen reader will read. I checked it and it's not clear to understand that these are two words. This could be a list (then user also get an info how many entries there are) as soon as it's more than one item. That way it's clear for SR users (and may also help other assistive technology).

Reply:
Yes, that was a little weird to me. This is how it is rendered in the DOM, even if visually they are separate `Chip` components. I think the `ChipContainer` might actually make them behave correctly. *However* `ChipContainer` seems to break the wrapping behaviour of cell contents, which has in pretty bad results visually (the individual chips inside the container do not wrap).

Will try a list ^^

Reply:
A list did not work. The only way I could have the values properly separated was by adding a space to the value passed as `text` to `Chip` [bb0ac03](https://github.com/aiven/klaw/pull/581/commits/bb0ac03fe5cf838801926500cb5a0fb10433ac26)

Reply:
In `TopicTable` I kind of cheated and used 

```
<li key={index + env}>
    <Chip text={env} />
</li>
```

🙈  

But we can also go with space-separated values for now and check that again later!